mungegithub/mungers/publish_scripts/publish_client_go.sh

```diff
@@ -52,7 +52,7 @@ trap cleanup_github_token EXIT SIGINT
 # this currently only updates commit hash of k8s.io/apimachinery
 update_godeps_json() {
     local godeps_json="./Godeps/Godeps.json"
-    local old_revs
+    local old_revs=""
```

Comment:
This change is added in the my last push. Without the change, line 70 will report "onbounded variable old_revs" if "while" loop at line 59 is skipped.